service-discovery/README.adoc

```diff
@@ -0,0 +1,324 @@
+====
+
+// Purpose statement for the assembly
+[role="_abstract"]
+As a developer of applications and services, you can connect Quarkus applications to Kafka instances in {product-long}. https://quarkus.io/[Quarkus^] is a Kubernetes-native Java framework made for Java virtual machines (JVMs) and native compilation, and optimized for serverless, cloud, and Kubernetes environments. Quarkus is designed to work with popular Java standards, frameworks, and libraries like Eclipse MicroProfile and Spring, as well as Apache Kafka, RESTEasy (JAX-RS), Hibernate ORM (JPA), Infinispan, Camel, and many more.
+
+OpenShift is an Enteprise Kuberentes Platform. When deploying Quarkus and any other type of Applications developers can use 
+https://developers.redhat.com/blog/2019/12/19/introducing-the-service-binding-operator/[ServiceBinding] capability that would let 
+their applications to discover configuration automatically inside their OpenShift or Kubernetes instance.
+
+RHOAS CLI contains set of commands that provide capability to connect and bind your OpenShift deployed applications with {product-long} services.
+You can review all possible commands in https://github.com/redhat-developer/app-services-cli/blob/main/docs/commands/rhoas_cluster.adoc[cli documentation].
+
+This quick start shows you how to connect Red Hat OpenShift Streams for Apache Kafka to OpenShift using the RHOAS CLI.
+    
+In this Quick Start, you'll connect your OpenShift Streams for Apache Kafka cloud service to your OpenShift cluster using the Red Hat OpenShift Application Services. (RHOAS) Operator and CLI. For more information on Red Hat OpenShift Application Services, please visit [cloud.redhat.com.](https://cloud.redhat.com/application-services)
+
+.Prerequisites
+ifndef::community[]
+* You have a Red Hat account.
+endif::[]
+* You have a running Kafka instance in {product}.
+
+// Condition out QS-only content so that it doesn't appear in docs.
+// All QS anchor IDs must be in this alternate anchor ID format `[#anchor-id]` because the ascii splitter relies on the other format `[id="anchor-id"]` to generate module files.
+ifdef::qs[]
+[#description]
+Using the {product-long} services in your OpenShift cluster{product-long}.
+
+[#introduction]
+Welcome to the {product-long} Service Discovery quick start. In this quick start, you'll learn how to connect your {product} services with https://quarkus.io/[Quarkus^]. Tutorial will connect Kafka with your Application to produce and consume messages from your Kafka instances in {product}.
+endif::[]
+
+[id="proc-installing-cli_{context}"]
+== Installing required cli tools
```

Comment:
Alternatively, instead of making this a procedure, you could add both of these CLI tools to the quick start prereqs (i.e. "The oc CLI is installed.").

Reply:
Amazing idea

Reply:
I done that and I feel that those can be better as procedure as we can use verification to check if they work etc.
If we have those as requirements people might be - ok I have OC installed and go with it.